packer/scripts/macos/macos-agentsetup.sh

```diff
@@ -13,29 +13,42 @@ sudo chown -R ec2-user:staff /var/jenkins
 /usr/local/bin/brew install grep
 /usr/local/bin/brew install wget 
 /usr/local/bin/brew install maven
+/usr/local/bin/brew install dpkg
 
-## Install MacPorts, jEnv, setup java8,11,17,21 and python, then set default java to 11
+## Array of JDK versions in form of version<space>path<space>priority
+jdk_versions=(
+    "8 /jdk8u392-b08/OpenJDK8U-jdk_x64_mac_hotspot_8u392b08.tar.gz 1"
```

Comment:
Should just use `@` as the separator instead of space, because sometimes urls can contain spaces. Then in `cut` just use `-d '@'` would be good.

Reply:
thanks for the comment... good point .. I did not think of the cases that some URLs might be having the spaces.. and the full URL path as well .. let me change to that ..